src/nls/it/strings.js

```diff
@@ -68,8 +68,8 @@ define({
     "ERROR_DELETING_FILE_TITLE"         : "Errore durante il tentativo di eliminazione del file",
     "ERROR_DELETING_FILE"               : "Errore durante il tentativo di eliminazione del file <span class='dialog-filename'>{0}</span>. {1}",
     "INVALID_FILENAME_TITLE"            : "Il nome del file non è valido",
```

Comment:
- `INVALID_FILENAME_TITLE` should become `Il {0} non è valido`
- `DIRECTORY_NAME` should become `nome della cartella`
- `FILENAME` should become `nome del file`